pinc/misc.inc

```diff
@@ -898,9 +914,21 @@ function is_valid_zip_file(string $file_path, bool $ignore_extension_check = fal
 
     if ($result === true) {
         $zip->close();
+    } else {
+        $zip_results = [
```

Comment:
should we define this as a global so we don't have to reconstruct it every error? I suppose it doesn't matter much.

Reply:
I don't know what is best.

Reply:
If you make it a static variable it is only constructed on first use, but I don't think we need that optimization here because the odds that this function will be used many times on a single page and return errors all of those times is low.